src/fontra/views/editor/editor.html

```diff
@@ -32,6 +32,13 @@
       --sidebar-tab-width: 3em;
       --sidebar-content-width-left: 20em;
       --sidebar-content-width-right: 22em;
+      --resize-cursor: col-resize;
+    }
+
+    :root.--resize {
```

Comment:
What's the reason for this particular naming? Couldn't this class simply be called "resizing"?